_articles/ms/metrics.md

```diff
@@ -0,0 +1,134 @@
+
+Sekiranya anda berminat untuk memahami projek anda pada tahap yang lebih mendalam, baca cara untuk menganalisis aktiviti projek anda.
+
+## Penemuan
+
+Sebelum ada yang dapat menggunakan atau menyumbang kembali ke projek anda, mereka perlu mengetahui bahawa ia wujud. Tanya pada diri anda: _apa orang mencari projek ini?_
+
+![Traffic graph](/assets/images/metrics/repo_traffic_graphs_tooltip.png)
+
+Sekiranya projek anda dihoskan di GitHub, [you can view](https://help.github.com/articles/about-repository-graphs/#traffic) berapa ramai orang yang datang ke projek anda dan dari mana asalnya. Dari halaman projek anda, klik "Wawasan", kemudian "Lalu Lintas". Di halaman ini, anda dapat melihat:
+
+* **Jumlah paparan halaman:** Memberitahu anda berapa kali projek anda dilihat
+
+* **Jumlah pelawat unik:** Memberitahu anda berapa orang yang melihat projek anda
+
+* **Merujuk laman web:** Memberitahu anda dari mana pengunjung datang. Metrik ini dapat membantu anda mengetahui di mana untuk menjangkau khalayak anda dan adakah usaha promosi anda berjalan lancar.
+
+* **Kandungan popular:** Memberitahu anda tempat pelawat pergi ke projek anda, dipecah mengikut paparan halaman dan pelawat unik.
+
+[GitHub stars](https://help.github.com/articles/about-stars/) juga dapat membantu memberikan ukuran populariti asas. Walaupun bintang GitHub tidak semestinya berkorelasi dengan muat turun dan penggunaan, mereka dapat memberitahu anda berapa banyak orang yang memperhatikan pekerjaan anda.
+
+Anda juga mungkin mahu [track discoverability in specific places](https://opensource.com/business/16/6/pirate-metrics): sebagai contoh, Google PageRank, lalu lintas rujukan dari laman web projek anda, atau rujukan dari projek atau laman web sumber terbuka yang lain.
+
+## Penggunaan
+
+Orang ramai mencari projek anda mengenai perkara liar dan gila yang kami panggil internet ini. Sebaik-baiknya, apabila mereka melihat projek anda, mereka akan merasa terdorong untuk melakukan sesuatu. Soalan kedua yang ingin anda ajukan ialah: _apa orang menggunakan projek ini?_
+
+Sekiranya anda menggunakan pengurus pakej, seperti npm atau RubyGems.org, untuk mengedarkan projek anda, anda mungkin dapat mengesan muat turun projek anda.
+
+Setiap pengurus pakej boleh menggunakan definisi "download" yang sedikit berbeza, dan muat turuntuk mengesan statistik penggunaan di banyak pengurus pakej yang popular.
+
+Sekiranya projek anda berada di GitHub, arahkan kembali ke halaman "Traffic". Anda boleh menggunakanun tidak semestinya berkaitan dengan pemasangan atau penggunaan, tetapi menyediakan beberapa asas untuk perbandingan. Cuba gunakan [Libraries.io](https://libraries.io/) to track usage statistics across many popular package managers.
+
+If your project is on GitHub, navigate again to the "Traffic" page. You can use the [clone graph](https://github.com/blog/1873-clone-graphs) to see how many times your project has been cloned on a given day, broken down by total clones and unique cloners.
```

Comment:
Incomplete translation?

Reply:
Ah, I've accidentally overlooked those short snippets. Terribly sorry. Just filled in the missing translated portions. Should be okay now.

Reply:
Do note that there are many words that the Malay language has adopted from English such as "popular" or words that are very closely similar such as "Analysis"(English) and "Analisis"(Malay). So it is totally normal for some of the words to look like typos of normal English words.

Reply:
I'm gonna go to bed first, it's 8:40am over here and I've stayed up all night doing this translation. @mlinksva Thanks for pointing out the many mistakes that I've made and point me in the right direction. I will probably continue translating this in another 4/5 hours from now.

Reply:
I went ahead and merged. Feel free to make a new PR if you'd like to further improve the translation. Thanks!